### ISSUE TITLE
Activation / focus issue (stealing focus)

### ISSUE THREAD

**Issue by cosote** [leapdroid-team/leapdroid#3]
When you have LeapDroid running not focused and send ADB commands like "input keyevent 3" to return to home and any app in android is running, like browser or play store or a game, LeapDroid Window gets activated and has focus. Also, this strange message is displayed in ADB session:
root@klte:/ # input keyevent 3
input keyevent 3
**WARNING: linker: libdvm.so has text relocations. This is wasting memory and is a security risk. Please fix.**

**Comment by LegendaryOverlord** [comment]
This happens to me anytime Leapdroid doesn't have focus, and something is loading (I think). For example, if I started a game up, and then went to type this, Leapdroid would steal focus multiple times. If I already have the game running, then it doesn't seem to happen.

**Comment by cosote** [comment]
QtGui4.dll is calling User32.DLL SetFocus for all different actions if LeapDroid doesn't have focus.
Would be great to remove those calls or so :)

**Comment by huisinro** [comment]
Thanks for reporting this issue. We noted the icon on taskbar will flash, but Leapdroid window still stays at the background, would this bother you? I don't see Leapdroid window comes at top, did you observe differently?

We will remove that warning message about text relocations.

**Comment by cosote** [comment]
Yes, it will popup on some Windows OS... I'm using Windows Server 2012. My current work-a-round is to use my own patched QT4 library or hook LeapDroid API calls and disable any call to SetForegroundWindow. QT4 patch in qwidget_win.cpp would be this (not so nice, I know ;) ...):
void QWidget::activateWindow()
{
    window()->createWinId();
    **//SetForegroundWindow(window()->internalWinId());**
}
I think LeapDroid is just calling some activateWindow()... just remove those and problem fixed.

**Comment by huisinro** [comment]
Hi Coste,

I noted that you did a patch in mybot.run website to remove the focus? can you post the patch here, and will incorporate it. Thanks,
